CMakeLists.txt

```
@@ -341,6 +341,11 @@ set(TS_USE_MALLOC_ALLOCATOR ${ENABLE_MALLOC_ALLOCATOR})
 set(TS_USE_ALLOCATOR_METRICS ${ENABLE_ALLOCATOR_METRICS})
 find_package(ZLIB REQUIRED)
 
+find_package(zstd)
+if(zstd_FOUND)
+  set(HAVE_ZSTD_H TRUE)
+endif()
```

This is probably fine, but it does leave `HAVE_ZSTD_H` unset if the condition is false. What do you think about the following?
```suggestion
set(HAVE_ZSTD_H ${zstd_FOUND})
```